.gitignore

```diff
@@ -30,3 +30,7 @@ node_modules
 .DS_Store
 
 build
+
+# Locks
+yarn.lock
+package-lock.json
```

Comment:
More of a question than a comment, but what is the outlook of upgrading `slay` to `npm@5`? In that case `package-lock.json` would need to be included.

Reply:
Would like to see this removed in a PR that adds a `.gitattributes` file to treat `package-lock.json` as a binary file.